src/spdx_tools/spdx3/model/dataset/dataset.py

```diff
@@ -0,0 +1,56 @@
+#  SPDX-FileCopyrightText: 2023 spdx contributors
+#
+#  SPDX-License-Identifier: Apache-2.0
+from enum import Enum, auto
+from typing import Dict, Optional
+
+from spdx_tools.common.typing.dataclass_with_properties import dataclass_with_properties
+from spdx_tools.common.typing.type_checks import check_types_and_set_values
+
+
+class ConfidentialityLevelType(Enum):
+    RED = auto()
+    AMBER = auto()
+    GREEN = auto()
+    CLEAR = auto()
```

Comment:
This one is not listed in the markdown file, but probably an issue in the spdx-3-model?

Reply:
Yes, I opened an issue for this.